modules/upgrade/pages/client-migration.adoc

```diff
@@ -1,184 +1,153 @@
 [[client-migration]]
 = Client Migration
 
+Upgrading traditional clients from earlier {slea}{nbsp}12 Service Packs to later {slea}{nbsp}12 Service Packs is fully automated.
+For example, upgrading from {slea}{nbsp}12{nbsp}SP1 to {slea}{nbsp}12{nbsp}SP4.
+This also applies for other supported products based on {slea}{nbsp}12.
 
+Upgrading traditional clients from {slea}{nbsp}12 with the latest Service Pack to {slea}{nbsp}15 is also automated, but you will need to do some preparation steps before you can upgrade.
 
+For supported {sle}{nbsp}15 upgrade paths, see  https://www.suse.com/documentation/sles-15/book_sle_upgrade/data/sec_upgrade-paths_supported.html
 
-Upgrading from SLE{nbsp}12 with the latest service pack (SP) to SLE{nbsp}15 can be automated, but requires some preparation steps.
-
-To upgrade the SP version on SLE{nbsp}12 (for example, upgrading from SLE{nbsp}12 or any SLE{nbsp}12{nbsp}SPx to SLE{nbsp}12{nbsp}SP4) can be fully automated and requires no additional preparation.
-
-
-
-[[bp.client.migration.sle]]
-== Upgrading SLE{nbsp}12{nbsp}SPx to Version 15
+[NOTE]
+====
+This process cannot be used to upgrade Salt clients
+====
 
-SLE{nbsp}12{nbsp}SPx clients can be auto-upgraded to SLE{nbsp}15 with {yast} auto-installation.
-This also applies for other supported products based on SLE{nbsp}12.
-For generally supported {sle} upgrade paths, see  https://www.suse.com/documentation/sles-15/book_sle_upgrade/data/sec_upgrade-paths_supported.html ({slea} {upgrade}, Chapter "`Supported Upgrade Paths to SLE 15`").
+// Do we have Salt instructions somewhere? LKB 2019-08-22
 
-It is important that you migrate the client to the latest available SP first.
-Upgrade to SLE 12 SP4 after December 2018.
 
-.Auto-Upgrading Salt Clients Currently Not Supported
-[IMPORTANT]
-====
-This procedure will work for traditionally  managed systems (system type [systemitem]``management``).
-It is not currently available for systems using Salt (system type [systemitem]``salt``).
-====
 
-During the procedure, the machine reboots and performs the system upgrade.
-The process is controlled by {yast} and {ay}, not by [command]``zypper`` commands.
+// We should probably split the SLE12->SLE12 and SLE12->SLE15 instructions. LKB 2019-08-22
```

Comment:
Here we only need SLE12->SLE15 (SP0) and a forward-looking remark about SLE15 (SP0) -> SLE15 (SP1) (and later).

Then we need short statements, how a user can migrate older clients (11 or 12) to a supported version (either LTSS or SLE 15).

Reply:
Yep, that makes sense. I'll open an issue to track this.

Reply:
https://github.com/SUSE/spacewalk/issues/9153